CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to redux-saga
+Contributing to redux-saga isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help better redux-saga.
+
+## Filing a bug
+When filing a bug, please follow the provided template:
+
+```
+### Description of the bug/issue

```

Comment:
Would be cool to mention that reproduction case helps a lot and if one is provided it's usually WAY quicker to fix the issue. Repros can be made as separate git repositories or created on http://codesandbox.io/